server/sonar-web/src/main/js/components/issue/Issue.js

```diff
@@ -29,7 +29,7 @@ import { updateIssue } from './actions';
 
 /*::
 type Props = {|
-  branch?: string,
+  branchLike?: { id?: string; name: string },
```

Comment:
Honestly I think it would bring much value. The name will give a hint about right type to use. Moreover we don't have a central place to store flow types.

Reply:
Ah yeah it would be a pain to import the flow type everywhere. Ok let's keep it like this, we will just need to be a bit careful!